xbmc/utils/FileUtils.cpp

```diff
@@ -249,6 +249,97 @@ CDateTime CFileUtils::GetModificationDate(const int& code, const std::string& st
   return dateAdded;
 }
 
+CDateTime CFileUtils::GetCreationDate(const std::string& strFileNameAndPath)
+{
+  CDateTime dateCreated;
+  if (strFileNameAndPath.empty())
+  {
+    CLog::Log(LOGDEBUG, "%s empty strFileNameAndPath variable", __FUNCTION__);
+    return dateCreated;
+  }
+
+  try
+  {
+    std::string file = strFileNameAndPath;
+    if (URIUtils::IsStack(strFileNameAndPath))
+      file = CStackDirectory::GetFirstStackedFile(strFileNameAndPath);
+
+    if (URIUtils::IsInArchive(file))
+      file = CURL(file).GetHostName();
+
+      // Try to get the creation datetime
+      // Use statx if available on posix to get the actual file creation time if possible
```

Comment:
> Use spaces as tab policy with an indentation size of 2.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#32-indentation

Reply:
Ah, I see it now (I think) the comment lines are indented